packages/app/src/sections/dashboard/FuturesHistoryTable.tsx

```diff
@@ -272,62 +271,64 @@ const FuturesHistoryTable: FC = () => {
 										</>
 									)
 								},
-								width: 60,
+								size: 60,
 							},
 							{
-								Header: () => (
+								header: () => (
 									<div>
 										<div>{t('dashboard.history.futures-history-table.side')}</div>
 										<div>{t('dashboard.history.futures-history-table.type')}</div>
 									</div>
 								),
-								accessor: 'side',
-								Cell: (cellProps: CellProps<FuturesTrade>) => {
+								accessorKey: 'side',
+								cell: (cellProps) => {
 									return conditionalRender(
 										cellProps.row.original.side,
 										<div>
-											<PositionType side={cellProps.value} mobile />
+											<PositionType side={cellProps.getValue()} mobile />
 											<div>{cellProps.row.original.orderType}</div>
 										</div>
 									)
 								},
-								width: 60,
+								size: 60,
 							},
 							{
-								Header: () => (
+								header: () => (
 									<div>
 										<div>{t('dashboard.history.futures-history-table.size')}</div>
 										<div>{t('dashboard.history.futures-history-table.price')}</div>
 									</div>
 								),
-								accessor: 'size',
-								Cell: (cellProps: CellProps<FuturesTrade>) => {
+								accessorKey: 'size',
+								cell: (cellProps) => {
 									return conditionalRender(
 										cellProps.row.original.price,
 										<div>
-											<div>{formatCryptoCurrency(cellProps.value, { suggestDecimals: true })}</div>
+											<div>
+												{formatCryptoCurrency(cellProps.getValue(), { suggestDecimals: true })}
+											</div>
 											<div>{formatDollars(cellProps.row.original.price ?? 0)}</div>
 										</div>
 									)
 								},
-								width: 60,
+								size: 60,
 							},
 							{
-								Header: <div>{t('dashboard.history.futures-history-table.pnl')}</div>,
-								accessor: 'netPnl',
-								Cell: (cellProps: CellProps<FuturesTrade>) => {
+								header: () => <div>{t('dashboard.history.futures-history-table.pnl')}</div>,
+								accessorKey: 'netPnl',
+								cell: (cellProps) => {
 									return conditionalRender(
-										cellProps.value,
-										cellProps.value.eq(wei(0)) ? (
+										cellProps.getValue(),
+										cellProps.getValue().eq(wei(0)) ? (
```

Comment:
I'll add this. However, the performance benefits will be very minimal, since in v8, react-table memoizes the value internally, so that future calls to `getValue` do not incur additional perf costs (it was the main reason they switched from the `value` property to the `getValue` method.

Reply:
ok makes sense